### DISCUSSION TITLE
JSON export

### DISCUSSION THREAD

**Discussion by methikasapukotana** [carbon-design-system/carbon#17778] [Help]
Hi,
Is there a way to export the Carbon/React component collection in JSON format ?

I'm looking to add a custom collection of Carbon React components in the Carbon UI Builder (https://builder.carbondesignsystem.com/) using a JSON file. 
For this purpose, the tool requires a JSON representing the component collection.

Any ideas or suggestions?

**Comment by adamalston** [comment]
Hi, do you have an example of what you're looking for?
